### ISSUE TITLE
Add docs to methods

### ISSUE THREAD

**Issue by gonzalocasas** [mcneel/pythonstubs#8]
Autocomplete is great, but it would be amazing if the documentation of the .NET methods could be added to the stubs as well.

**Comment by ZacZhangzhuo** [comment]
agree. if there are more prompts of multiple constructors and descriptions that would be great. simply add more comments in each definition. This will help us A LOT to code faster and know Rhino Geometry better! @sbaer @BobMcNeel
